languages.yaml

```diff
@@ -2391,6 +2391,39 @@ Svelte:
       - "#FF3C00"
       - "#FFFFFF"
     chip: "#FF3E00"
+Svg:
+  type: data
+  ascii: |              
+    {0}                 ......                 
+    {0}               ....{1}--{0}....               
+    {0}        ....  ...{1}------{0}...  ....        
+    {0}     ............{1}------{0}............     
+    {0}    ...{1}-----{0}......{1}----{0}.....{1}------{0}..     
+    {0}    ...{1}------{0}.....{1}----{0}....{1}-------{0}...    
+    {0}    ...{1}--------{0}...{1}----{0}...{1}-------{0}...     
+    {0}     .....{1}-------{0}.{1}----{0}.{1}-------{0}.....     
+    {0}  ..........{1}----------------{0}..........  
+    {0} ...{1}---{0}.......{1}------------{0}.......{1}---{0}... 
+    {0}...{1}----------------------------------{0}...
+    {0}...{1}----------------------------------{0}...
+    {0} ...{1}---{0}.......{1}------------{0}......{1}----{0}... 
+    {0}  ..........{1}----------------{0}..........  
+    {0}     .....{1}-------{0}.{1}----{0}.{1}-------{0}.....     
+    {0}    ...{1}--------{0}...{1}----{0}...{1}-------{0}...     
+    {0}    ...{1}-------{0}....{1}----{0}....{1}-------{0}...    
+    {0}    ...{1}------{0}.....{1}----{0}.....{1}------{0}..     
+    {0}     ............{1}------{0}............     
+    {0}        ....  ...{1}------{0}...  ....        
+    {0}               ....{1}--{0}....               
+    {0}                 ......                 
```

Comment:
Great design, very clean. 

It's almost perfectly symmetrical:

![image](https://github.com/o2sh/onefetch/assets/13710835/c8352ab6-69c6-4c06-a9b0-7349fcecb52d)